src/main/java/JavaQuestionThree/Dfs.java

```diff
@@ -0,0 +1,38 @@
+package JavaQuestionThree;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+
+public class Dfs {
+
+
+    public void dfsUtil(int v, boolean visited[], ArrayList<Integer>[] depGraph, ArrayList<Integer> resultofDFS, int level)
```

Comment:
If this class contain only util functions make it a final class and make methods static.
Function name should be meaningful and should signify it's usage.